helm/alfresco-process-application/values.yaml

```diff
@@ -7,6 +7,9 @@ global:
   # global.registryPullSecrets -- Configure pull secrets for all deployments
   registryPullSecrets:
     - "quay-registry-secret"
+    - "aps2-registry-secret"
```

Comment:
do we want to add this additional registry secret for aps2-registry, as it can be added by Customer as per their need.

Reply:
this field will be normally overriden completely by the users, if the users want to use the cluster registry.. in this case will be already provided